commercetools-sdk-base/src/main/java/io/sphere/sdk/search/model/FilteredFacetExpressionImpl.java

```diff
@@ -25,4 +25,9 @@ public boolean equals(Object o) {
     public String alias() {
         return super.alias();
     }
+
+    @Override
```

Comment:
Probably the case is to enable the highlighting of the method in the developer's IDE. Otherwise the method (which should be an important one) is relegated to a less important role and developers might have trouble finding it. If that's not the case, then it's a mistake :) You can try it out in the tests.